testing/dart/plugin_utilities_test.dart

```diff
@@ -9,8 +9,10 @@ import 'package:test/test.dart';
 typedef StringFunction = String Function();
 typedef IntFunction = int Function();
 
+@pragma('vm:entry-point', 'get')
 String top() => 'top';
 
+@pragma('vm:entry-point')
```

Comment:
Shouldn't this be
```
class Foo {
  @pragma('vm:entry-point')
  static int getInt();
}
```

as we use the `getInt` in the plugin API which will cause C code to look up `Foo` and inside that the `getInt` method.
```
PluginUtilities.getCallbackHandle(Foo.getInt)
```
below? (if marking method as entrypoint doesn't imply class as entrypoint we may need both)

Reply:
Hmm, it ran fine in my local test with only the class annotation, which is odd given that you're right, that should be needed if the method is retrieved as well. I'll add both just to be sure here.

Reply:
Just as a followup on this, it's because `CallbackCache::LookupDartClosure` uses `Dart_GetClass` to get the class from the cached name, but then uses `Dart_GetStaticMethodClosure`, which goes through `Class::LookupFunctionAllowPrivate` (which does not do the entry point check, unlike `Dart_GetField` -> `Class::InvokeGetter`).

Reply:
> `Dart_GetStaticMethodClosure` which goes through ... which doesn't do the entrypoint check.

But we should dot the entrypoint check for `Dart_GetStaticMethodClosure` as well, right?

Reply:
Yes, `Dart_GetStaticMethodClosure` should probably just go through `Dart::InvokeGetter` (which does the check), maybe by just delegating the lookup to `Dart_GetField`,  instead of short-circuiting to `Class::LookupStaticFunctionAllowPrivate` (which is faster, because we won't look for a field first and then fall back to looking for a method, but is used by a lot more of the Dart VM internals and so can't be gated on entry points).